service/src/main/java/bio/terra/cbas/dependencies/common/DependencyUrlLoader.java

```diff
@@ -58,28 +59,28 @@ public String load(DependencyCacheKey key) throws DependencyNotAvailableExceptio
             .build(loader);
   }
 
-  private String fetchWdsUrl() throws DependencyNotAvailableException {
+  private String fetchWdsUrl(String userToken) throws DependencyNotAvailableException {
```

Comment:
The reason userToken is a `String` instead of `BearerToken` here and below is because the only places these 2 methods are called are from the CacheLoader and in the cache we only store access tokens as Strings. Instead of changing the data type of the token in Cache I left it as it is. I am open to the possibility of changing the data type of token in Cache to BearerToken as well.